From 554afbb6f95a5d218d4aea80dcf504f4e17c5f40 Mon Sep 17 00:00:00 2001
From: Aurelian Zanoschi <aurelian.zanoschi@freescale.com>
Date: Tue, 11 Dec 2012 18:31:11 +0000
Subject: [PATCH 087/518] dpa_mac: Add support for get_mac_handle

Added missing get_mac_handle in setup_memac function.

Signed-off-by: Aurelian Zanoschi <aurelian.zanoschi@freescale.com>
[Grabbed from the branch, LINUX_IR5.2.0, of
https://git.freescale.com/git-private/cgit.cgi/ppc/alu-b4860/linux.git.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 drivers/net/ethernet/freescale/dpa/mac-api.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/net/ethernet/freescale/dpa/mac-api.c b/drivers/net/ethernet/freescale/dpa/mac-api.c
index 7a16880..95e2f51 100644
--- a/drivers/net/ethernet/freescale/dpa/mac-api.c
+++ b/drivers/net/ethernet/freescale/dpa/mac-api.c
@@ -877,6 +877,7 @@ static void __devinit __cold setup_memac(struct mac_device *mac_dev)
 	mac_dev->change_addr    = change_addr;
 	mac_dev->set_multi      = set_multi;
 	mac_dev->uninit		= uninit;
+	mac_dev->get_mac_handle		= get_mac_handle;
 	mac_dev->fm_rtc_enable		= fm_rtc_enable;
 	mac_dev->fm_rtc_disable		= fm_rtc_disable;
 	mac_dev->fm_rtc_get_cnt		= fm_rtc_get_cnt;
-- 
1.7.5.4

